src/web/routes.rs

```diff
@@ -49,6 +49,10 @@ pub(super) fn build_axum_routes() -> AxumRouter {
             "/favicon.ico",
             get_static(|| async { Redirect::permanent("/-/static/favicon.ico") }),
         )
+        .route(
+            "/-/static/*path",
```

Comment:
This route seems weird - did you mean `:path`?

Reply:
this is a [wildcard capture](https://docs.rs/axum/latest/axum/struct.Router.html#wildcards), and does the same as `*` in the old router, without the need for `:single`